doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_debugging.rst

```diff
@@ -29,12 +29,16 @@ Set the following Kconfig options to ``y`` for the images running on the cores y
 Debug configurations
 ********************
 
+.. to review
+
 Some applications and samples provide a specific configuration that enables additional debug functionalities.
 You can select custom configurations when you are :ref:`configuring the build settings <cmake_options>`.
 
 Debugging multiple cores
```

Comment:
For debugging single core application (i.e. cpuapp) you can just use `west debug` which will start GDB session. This is a way to go before using Jlink scripts described below (which can be used to access any core but they are complex). I'm not sure how it works with sysbuild and what are the commands to debug child application, probably @tejlmand knows.

Reply:
Sysbuild does not apply to this release